plover/assets/main.json

```diff
@@ -64840,7 +64604,6 @@
 "PORPB/TKPWRAEF/EUBG": "pornographic",
 "PORPB/TKPWRAEFBG": "pornographic",
 "PORPB/TKPWRAF/EUBG": "pornographic",
-"PORPB/TKPWRAFBG": "important graphic",
```

Comment:
Pornographic already has the outline `PORPB/TKPWRAEFBG`, do you still want the one with `*`?"